server/server.go

```diff
@@ -298,6 +298,10 @@ func setSSLVariable(ca, key, cert string) {
 	variable.SetSysVar("ssl_ca", ca)
 }
 
+func setTxnScope() {
+	variable.SetSysVar("tidb_txn_scope", config.GetGlobalConfig().TxnScope)
+}
+
```

Comment:
Is this really needed?

Reply:
If we didn't set it, the default value of `tidb_txn_scope` would be the default value of `TxnScope`